homeassistant/components/demo/number.py

```diff
@@ -0,0 +1,92 @@
+"""Demo platform that offers a fake Number entity."""
+import voluptuous as vol
+
+from homeassistant.components.number import NumberEntity
+from homeassistant.const import DEVICE_DEFAULT_NAME
+
+from . import DOMAIN
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up the demo Number entity."""
+    async_add_entities(
+        [
+            DemoNumber(
```

Comment:
Maybe we should add a demo entity with a different step?

Reply:
Done